db/migrate/20190315170837_add_component_id_to_certificates.rb

```diff
@@ -0,0 +1,5 @@
+class AddComponentIdToCertificates < ActiveRecord::Migration[5.2]
+  def change
+    add_column :certificates, :component_id, :integer
```

Comment:
I think we want to use a 'reference' here. The syntax is [here](https://api.rubyonrails.org/v5.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)

Reply:
Also, we should make sure this can't be null.